pkg/forwarder/forwarder.go

```diff
@@ -39,6 +39,10 @@ type Config struct {
 	PodNetwork   *tunneler.Config `json:"pod-network"`
 	PodNamespace string           `json:"pod-namespace"`
 	PodName      string           `json:"pod-name"`
+
+	TLSServerKey  string `json:"tls-server-key,omitempty"`
+	TLSServerCert string `json:"tls-server-cert,omitempty"`
+	TLSClientCA   string `json:"tls-client-ca,omitempty"`
```

Comment:
nit: gofmt ?

Reply:
Could you  explain in a little more detail? `gofmt` doesn't show anything in my environment.

```
gofmt -s -d pkg/forwarder/forwarder.go
```

Reply:
My bad. Please ignore